src/views/HousingLottery/index.jsx

```diff
@@ -1,7 +1,24 @@
+import { Button, Card, CardContent, CardHeader, Grid, List } from '@mui/material';
+import PreferredHallsCard from './studentView/PreferredHall';
 import PreferredHall from './studentView/PreferredHall';
+import StudentApplicants from './studentView/StudentApplicants/index.jsx';
+
+
 
 const HousingLottery = () => {
-  return <PreferredHall />;
+
+
+
+  return (
+    <Grid container spacing={2} justifyContent="center">
+      <Grid item xs={12} lg={10}>
+        <PreferredHallsCard/>
```

Comment:
Is there a reason you remove `<PreferredHall />` and put a `<PreferredHallsCard/>` here? I didn't see you define `PreferredHallsCard` anywhere, but I'm surprised that in vm it still loads the Preferred Hall section.

Reply:
I think I accidentally changed that after trying couple of things. I'll change it back to <PreferredHall />!!